eth2/api/http/ir.py

```
@@ -129,13 +113,12 @@ def get_block_proposal(
             slot, parent_block_root, randao_reveal, parent_state, state_machine
         )
 
-    async def broadcast_block(self, signed_block: SignedBeaconBlock) -> bool:
+    async def broadcast_block(self, signed_block: SignedBeaconBlock):
```

we will want to signal to the client if broadcast fails -- we can throw an exception here but will need a way to transmit that across the HTTP boundary -- food for thought.